contrib/docker/docker-compose-postgres/README.md

```diff
@@ -0,0 +1,31 @@
+# Docker Compose Development with Postgres
+
+This directory includes files to help you get running with Postgres locally.
+
+## Usage
+
+1. Copy these files into your project
+2. Update `app-config.development.yaml` to use Postgres
```

Comment:
This isn't gonna work now that https://github.com/spotify/backstage/pull/2994 is in, but we could have a separate config file for postgres and something like a `"start-backend-pg": "yarn start-backend --config ../../app-config.yaml --config postgres-config.yaml"` script in the backend